tests/cache/jobQueue/getHierarchy.js

```diff
@@ -127,51 +127,15 @@ describe('tests/cache/jobQueue/getHierarchy.js, ' +
         }
 
         nonWorkerResponse = res.body;
+        tu.toggleOverride('enableWorkerProcess', true);
```

Comment:
need to save the initial state of the toggles in a *before* block, then do the toggle override in the before block, AND set it back to the initial value in an *after* block -- otherwise we risk having a test fail and never getting to the code which changes the state back

Reply:
This is in a before block, and it's already being set back to false in an after block on line 286.
Are you saying we should be saving the initial value instead of assuming it was false?

Reply:
we should not assume it was false

Reply:
ok. updated.